src/components/Chip/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import CloseIcon from './closeIcon';
+import ButtonIcon from './../ButtonIcon';
+import RenderIf from '../RenderIf';
+import './styles.css';
+
+/**
+* A Chip displays a label that can be removed from view.
+*/
+export default function Chip(props) {
+    const {
+        label,
+        onDelete,
+        variant,
+        className,
+        style,
+    } = props;
+
+    const getVariantClassNames = () => {
+        if (variant === 'base') {
+            return null;
+        }
+        return `rainbow-chip--${variant}`;
+    };
+
+    const getContainerClassName = () => classnames('rainbow-chip', getVariantClassNames(), className);
+
+
+    return (
+        <span className={getContainerClassName()} style={style}>
+            {label}
+            <RenderIf isTrue={!!onDelete}>
+                <ButtonIcon
+                    className="rainbow-chip_button-container"
+                    icon={<CloseIcon />}
+                    size="small"
+                    title="Close"
+                    onClick={onDelete}
+                    assistiveText="Remove" />
+            </RenderIf>
+        </span>
+    );
+}
+
+Chip.propTypes = {
+    /** The title that appears in the Chip. */
```

Comment:
look at badge description, title is the tooltip showed when hover.
also I think that we should add title prop like badge